content/administration/upgrade.rst

```diff
@@ -1,264 +1,314 @@
+    - Sell this product (only available with Sales App) to a random customer;
+    - Open your customer database (Contact App), select a random customer (or company) and double-check its data;
+    - Ship this product (only available with Inventory App);
+    - Check if the route to ship this product applies the same set in your production database (only available with Inventory App);
+    - Validate a customer invoice (only available with Invoicing and/or Accounting Apps);
+    - Credit the invoice (issue a credit note) and check if its behaves as your production database;
+    - Check your Reports results (only available with Accounting Apps);
+    - Randomly check your taxes, currencies, Bank Account. Is your fiscal year set in production database the same? (only available with Accounting Apps);
+    - Proceed to an online order (only available with Website Apps) from the product selection in your shop until the checkout process and check if its behaves as your production database.
 
-The production upgrade request is when you decide to upgrade your current database with all your
-production data (invoices, VAT returns, inventories, current orders) to a new version of your
-choice.
+    Depending on the complexity of your database, you also shouldn't forget to test :
 
-After your :ref:`tests <upgrade/testing-phase>` are completed to your satisfaction, submit the
-request to upgrade your production database via our `website form <https://upgrade.odoo.com>`_.
-Select *Production* purpose.
+    - Integrations with external softwares (EDI, APIs, ...)
+    - Workflows between different Apps (online sales with eCommerce, converting a lead all the way to a sales order, delivery of products, etc ... )
+    - Exporting data
+    - Your automated actions to make sure they work
+    - Your server actions in the Action menu on form views as well as by selecting multiple records on list views
+
+    Those are non-exhaustive lists that you can extend to your other Apps based on your use of Odoo.
+
+In case of an issue with your test database, you can request the assistance of Odoo via the `support page <https://odoo.com/help>`_. They will be able to provide guidance on your upgrade or fix the issue. In any case, it is important to report any issue you encounter during your testing to Odoo so that they can be fixed before the upgrade of your production database.
+
+Upgrading your production database
+----------------------------------
+
+Once you completed your :ref:`tests <upgrade/test_your_db>` and are confident that you can use your upgraded database as your main database without any issue, it is time to plan the Go-live day. During the upgrade of your production database, any modification done on it will not be saved. This is why we recommend not using your database during that time.
+
+The Go-live day can be planned in coordination with the upgrade support analysts of Odoo reachable via the `support page <https://odoo.com/help>`_.
 
 .. important::
```

Comment:
What about merging the 2 paragraphs into this `Once the :ref:`tests <upgrade/test_your_db>` are completed, and you are confident that the upgraded database can be used as your main database without any issue, it is time to plan the go-live day. It can be planned in coordination with Odoo's upgrade support analysts, reachable via the `support page <https://odoo.com/help>`_.` ? (it will be in my commit so you'll be able to review at that time :D